### DISCUSSION TITLE
Supporting url modules?

### DISCUSSION THREAD

**Discussion by reggi** [oven-sh/bun#1217] [General]
I'm curious if bun has any plans to support url modules in some way either as deno has, or via the web spec.

**Comment by Jarred-Sumner** [comment]
Bun will support it. I don't know about specific timeline, but I think not supporting it would be a mistake. 

`bun install` still is completely separated from the runtime, which is kind of silly. There's no good reason to ask people to run `bun install` when dependencies are already installed globally
